src/tree_cmd.cpp

```diff
@@ -31,6 +32,8 @@
 #include "table/clear_land.h"
 
 #include "safeguards.h"
+#include <array>
+#include <cstdint>
```

Comment:
Oh, you don't need to add these, they are already included via stdafx.h

Reply:
oh! i'll remove them, then.